### ISSUE TITLE
ImportError:  attempted relative import with no known parent package

### ISSUE THREAD

**Issue by ranlo** [gologinapp/pygologin#86]
- cloned the repo
- python3.12 setup.py install
- cd gologin
- python3.12 gologin-selenium.py

getting an error:

ImportError:  attempted relative import with no known parent package

caused by

from .extensionsManager import ExtensionsManager

**Comment by stepjet** [comment]
try to install gologin via pip

**Comment by bbajwa346** [comment]
> try to install gologin via pip

NO,that doesn't resolve anything issue is persistant !! I was using it 6 months ago but in last updates there is some issue caused by 'from .extensionsManager import ExtensionsManager'   Kindly check and resolve the issue as soon as possible

**Comment by stepjet** [comment]
could you try in with this version https://pypi.org/project/gologin/2024.5.12073431/

**Comment by bbajwa346** [comment]
> could you try in with this version https://pypi.org/project/gologin/2024.5.12073431/

Tried still same error '

It doesn't even let create profile

> could you try in with this version https://pypi.org/project/gologin/2024.5.12073431/

'

**Comment by igortikushEasync** [comment]
The error ImportError: attempted relative import with no known parent package happens because:
The code is using relative imports (the . in from .extensionsManager import ExtensionsManager)
Use python3 -m gologin.gologin-selenium
The -m flag tells Python to run the file as a module, which properly sets up the Python package structure and allows relative imports to work.
